app/api.py

```diff
@@ -21,8 +21,8 @@ def start_response(color):
     )
 
 def move_response(move):
-    assert move in ['up', 'down', 'left', 'right'], \
-        "Move must be one of [up, down, left, right]"
+#    assert move in ['up', 'down', 'left', 'right'], \
```

Comment:
Are we sure about getting rid of this? Is this for better passthrough of errors to the unit test output? Maybe we should comment the purpose of removing it so we remember the reasoning.

Reply:
err idk that was you on my computer /shrug